CMakeLists.txt

```diff
@@ -24,7 +24,7 @@
 #
 #=============================================================================
 
-cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
```

Comment:
What is the CMake shipped with the oldest still freely supported Ubuntu LTS or Debian Stable?

Reply:
* For Ubuntu (20.04 if considering up to standard LTS): 3.16.
* For Debian (Bullseye if considering LTS): 3.18.

So it should be fine?